.gitlab-ci.yml

```diff
@@ -0,0 +1,20 @@
+# To contribute improvements to CI/CD templates, please follow the Development guide at:
+# https://docs.gitlab.com/ee/development/cicd/templates.html
+# This specific template is located at:
+# https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/gitlab/ci/templates/Terraform-Module.gitlab-ci.yml
+
+include:
```

Comment:
The pipelines are currently running on Github actions. Could you please remove this?

Reply:
I'll kick of the pipeline for this changes

Reply:
I did this in a hurry. I'll finish my work a submit a larger PR with proper care.

Reply:
@yves-vogl no worries let us know if you have any questions